fdbclient/BlobGranuleFiles.cpp

```diff
@@ -1878,7 +1877,7 @@ const EncryptCipherBaseKeyId encryptBaseCipherId = deterministicRandom()->random
 const EncryptCipherRandomSalt encryptSalt = deterministicRandom()->randomUInt64();
 
 Standalone<StringRef> getBaseCipher() {
-	Standalone<StringRef> baseCipher = makeString(AES_256_KEY_LENGTH);
+	Standalone<StringRef> baseCipher = makeString(deterministicRandom()->randomInt(4, 256));
```

Comment:
Other places to potentially modify:

https://github.com/apple/foundationdb/blob/main/fdbserver/IPager.cpp#L37

https://github.com/apple/foundationdb/blob/main/fdbserver/workloads/EncryptionOps.actor.cpp#L551

Reply:
Thanks for pointing out, Nim. 

Done.